Slugbotics/ARM
# Aaron vison yolov4 tiny

This adds two things:
1) a laptop hal, this is for getting the laptop's webcam input for testing
2) a new object identifier powered by YOLO V4 tiny

To Test:

unfortunately this is not supper easy to test, you need to change a few things in main, I was going to include this in the PR, but I dont want to apply these changes to main, they are just needed to test the model

1) apply the laptop HAL at the end of HAL stuff (ln.172)
+from HALs.laptop_test_HAL import laptop_HAL
+selected_HAL = laptop_HAL()

2) apply the Object Identifier at the  # vision stuff area
-selected_object_identifier: VisualObjectIdentifier = ColorObjectIdentifier()
+from Vision.yolov4_tiny.YoloV4TinyObjectIdentifier import YoloV4TinyObjectIdentifier
+selected_object_identifier: VisualObjectIdentifier = YoloV4TinyObjectIdentifier()

3) Make the controller actually use a label from the classes.txt file that you think your webcam will be able to see, I selected "Person" because I am a person and I can make myself be in frame at the  # controller stuff area
-selected_controler: Controller = FollowLargestObjectControler(selected_HAL, selected_object_identifier, "Blue object")
+selected_controler: Controller = FollowLargestObjectControler(selected_HAL, selected_object_identifier, "Person")

it should show some frames of it identifying you, if so this PR is good!
